pkg/tanka/load_test.go

```diff
@@ -79,6 +79,36 @@ func TestLoad(t *testing.T) {
 
 		{
 			name:    "inline",
+			baseDir: "./testdata/cases/withenv/",
+			expected: manifest.List{{
+				"apiVersion": "v1",
+				"kind":       "ConfigMap",
+				"metadata": map[string]interface{}{
+					"name":      "config",
+					"namespace": "withenv",
+				},
+			}},
+			env: &v1alpha1.Environment{
+				APIVersion: v1alpha1.New().APIVersion,
+				Kind:       v1alpha1.New().Kind,
+				Metadata: v1alpha1.Metadata{
+					Name:      "withenv",
+					Namespace: "cases/withenv/main.jsonnet",
```

Comment:
the `StaticLoader` needs modification as well, it should be consistent with this behavior here

Reply:
That has bigger implications as it uses `ParseDir`, I made a commit, see what you make of it.